app/views/layouts/mailer.text.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
since you have layout for emails in text format maybe it's also worth adding emails content as well ;)